src/SlnParser.Tests/IntegrationTests.cs

```diff
@@ -327,6 +327,78 @@ public void Should_Be_Able_To_Parse_TestSln_Solution_Correctly()
                 .Contain(file => file.Name == "testNested1.txt");
         }
 
+        [Fact]
+        public void Parse_WithProjectWithoutPlatform_IsParsedCorrectly()
+        {
+            string solutionFilePath = GetTempFileName(".sln");
```

Comment:
There are multiple places where you don't use `var` according to the overall code-style. This is also configured in the `.editorconfig`

Reply:
I'm wondering if `.editorconfig` should be updated so that a compiler warning is produced for code which does not adhere to the coding style during code analysis?

Thank you.

Reply:
I'll think about that.